shell/platform/embedder/embedder.h

```diff
@@ -2510,6 +2510,25 @@ FLUTTER_EXPORT
 FlutterEngineResult FlutterEngineScheduleFrame(FLUTTER_API_SYMBOL(FlutterEngine)
                                                    engine);
 
+//------------------------------------------------------------------------------
+/// @brief      Schedule a callback to be called after the next frame is drawn.
+///             This callback is made on an internal engine managed thread and
+///             embedders must re-thread if necessary. Performing blocking calls
+///             in this callback may introduce application jank.
+///
+/// @param[in]  engine     A running engine instance.
+/// @param[in]  callback   The callback to execute.
+/// @param[in]  user_data  A baton passed by the engine to the callback. This
+///                        baton is not interpreted by the engine in any way.
+///
+/// @return     The result of the call.
+///
+FLUTTER_EXPORT
+FlutterEngineResult FlutterEngineSetNextFrameCallback(
```

Comment:
Documentation: Clarify that FlutterEngineSetNextFrameCallback must be made on the platform thread (the thread on which the FlutterEngineRun call was made).

Reply:
Documentation: Clarify if the callback is invoked repeatedly or only for one next frame.

Reply:
Ah, I was under the impression that all embedder APIs had to be called from the platform thread. Will add, thanks!

Reply:
Added